HHS/Head-Start-TTADP
# [PROD] Spelling correction, auto-db backup, postgres upgrade to 15

## Description of change

[TTAHUB-2905] Update text "Competetive" to "Competitive"
[TTAHUB-3003] Update awscli_install.sh to version lock awscli 
[TTAHUB-1828] Change all "GROUP BY TRUE" to "GROUP BY 1=1" to support postgres 15.x

## How to test
[TTAHUB-2905]
 * Go to RTR Profile page, find recipient grant with "below competitive" finding.
[TTAHUB-3003]
 * Run db backup script, awscli should download and install without failing the hash verification.
[TTAHUB-1828]
* tests work, can create a report
## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2905
* https://jira.acf.gov/browse/TTAHUB-3003
* https://jira.acf.gov/browse/TTAHUB-1828


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
